src/components/firstProfile/Status.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useState } from 'react'
+import { Typography, Box, Button } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import { commonStyles } from 'styles/commonStyles'
+
+const STATUSES: Array<string> = [
+  'Looking for new friends',
+  "I’m learning a new language. Let's talk!",
+  "Let's be friends, I'm new in town",
+  'I look for emotional support',
+  'My plans are to move abroad',
+  "I'm a new mom. Need some help",
+  "Let's take the dogs for a walk",
+]
+
+const Status = () => {
+  const { classes } = useStyles()
+  const commonClasses = commonStyles().classes
+
+  const [selectedStatuses, setSelectedStatuses] = useState<string[]>([])
+
+  const toggleStatus = (status: string) => {
+    if (selectedStatuses.includes(status)) {
+      setSelectedStatuses(selectedStatuses.filter((s) => s !== status))
+    } else {
```

Comment:
refactor condition plz, i believe you don't need all of them

Reply:
Could you please elaborate which on we do not need? 
We should check if the chosen status exists in the selectedStatuses array (line  23)
if it does, we filter the array and exclude this status (line 24)
otherwise we check the length of the array (wether the user has already chosen all 3 possible statuses, line  25-26) and if so we update the array (27)

Reply:
in `else` condition you have another `if`, test using of the condition `else if` instead.

Reply:
- [x] changed into if else